documentation/slate/source/includes/apis.md

```diff
@@ -591,58 +707,66 @@ The other one, the most used, is to access the 'journey' api endpoint:
     system).
 </aside>
 
+####Requesting a single journey
+
+The most used way to access to this service is to get the `/journeys` api endpoint.
+Here is the structure of a standard journey request:
+
+<https://api.navitia.io/v1/journeys?from={resource_id_1}&to={resource_id_2}&datetime={date_time_to_leave}> .
+
+<a
+    href="http://jsfiddle.net/gh/get/jquery/2.2.2/CanalTP/navitia/tree/documentation/slate/source/examples/jsFiddle/journeys/"
+    target="_blank"
+    class="button button-blue">
+    Code it yourself on JSFiddle
+</a>
+
 In the [examples](#examples), positions are given by coordinates and no network is specified.
 However when no coordinates are provided, you need to provide on what region you want to request as
 <https://api.navitia.io/v1/coverage/us-ca/journeys?from=-122.4752;37.80826&to=-122.402770;37.794682>
 
 The list of regions covered by navitia is available through [coverage](#coverage).
 
-
 <aside class="notice">
     If you want to use a specific data set, use the journey api within the
     data set: https://api.navitia.io/v1/coverage/{your_dataset}/journeys
 </aside>
 
-<aside class="success">
-    Neither the 'from' nor the 'to' parameter of the journey are required,
-    but obviously one of them has to be provided.
-    <br>
-    If only one is defined an isochrone is computed with every possible
-    journeys from or to the point.
-</aside>
+####Requesting an isochrone
+
+If you want to retrive every possible journeys form a single point at a time, you can request as:
```

Comment:
`retrieve` every possible `journey` `from` a single point at a time, you can request as `follow`: